### ISSUE TITLE
Fix regression in keto-k8 due to cloud-provider change

### ISSUE THREAD

**Issue by lewismarshall** [UKHomeOffice/keto-k8#49]
`kmm save-assets` now crashes with nil pointer ref as the struct has changed in the CloudProvider.

Fix!

**Comment by KashifSaadat** [comment]
Error from the `etcd-member` service:
```
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal systemd[1]: Starting etcd (System Application Container)...
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal rkt[4568]: rm: unable to resolve UUID from file: open /var/lib/coreos/etcd-member-wrapper.uuid: no such file or directory
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal rkt[4568]: rm: failed to remove one or more pods
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal grep[4579]: /dev/xvde /data ext4 rw,seclabel,relatime,data=ordered 0 0
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal docker[4582]: level=info msg="Cloud Provider Initialized ["aws"]"
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal docker[4582]: panic: runtime error: invalid memory address or nil pointer dereference
Jul 05 16:13:08 ip-172-30-2-21.eu-west-1.compute.internal docker[4582]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1c0ffb8]
```

**Comment by lewismarshall** [comment]
Fixed by #48
